goshopify.go

```diff
@@ -281,6 +285,9 @@ func CheckResponseError(r *http.Response) error {
 					topicAndElem := fmt.Sprintf("%v: %v", k, elem)
 					responseError.Errors = append(responseError.Errors, topicAndElem)
 				}
+			} else if reflect.TypeOf(v).Kind() == reflect.String {
+				topicAndElem := fmt.Sprintf("%v: %v", k, v)
+				responseError.Errors = append(responseError.Errors, topicAndElem)
```

Comment:
What is this part? It looks like this is the reason Codecov is complaining about coverage. Could you add a test that covers this use case?